x-pack/plugin/autoscaling/src/main/java/module-info.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
```

Comment:
This is the wrong license header - it mentions SSPL which is _not_ an alternative for X-Pack files.

It should be the same license header as the other file in this PR.